docs/source/index.md

```diff
@@ -1,86 +1,62 @@
 
-Then, create some boilerplate code for your new workflow.
+Lacth SDK allows developers to upload workflows to the full-featured [Latch Platform](console.latch.bio) with ease. The platform is built with Kubernetes, ensuring containerization, portability, and scalability are available out-of-the-box. Behind the scene, Latch takes advantage of AWS spot instances, offering ultra-fast runtimes with extremely low cloud costs for teams.
 
-```bash
-latch init testworkflow
-```
+<b>Bioinformaticians need to create an intuitive user interfaces for biologists.</b> As a lab or R&D team grows, the number of biologists per bioinformatician increases to a point where it's no longer sustainable for the bioinformatician to manage requests and run pipelines manually from her computer. Automating pipeline runs returns valuable time to the bioinformatician to focus on writing analyses to generate scientific insights. 
 
-Finally register the boilerplate code to [LatchBio](https://latch.bio).
+With Latch SDK, developers can write the description to their workflow and customize input parameters using plain Markdown. Latch automatically parses the written text and Python function headers to compile a type-safe UI. 
 
-```bash
-latch register testworkflow
-```
+<b>Specifying arbitrary cloud compute and storage resources for bioinformatics pipelines is difficult.</b> With Latch SDK, there are several Python task decorators that easily allow you to define the resources available at runtime. The framework starts at 2 CPUs and 4 GBs of memory and goes all the way to 31 CPUs, 120 GBs of memory and 1 GPU (24 GBs of VRAM, 9,216 CUDA cores) to easily handle all processing needs.
 
-This might take 3-10 minutes depending on your network connection. (Subsequent
-registers will complete in seconds by reusing the image layers from this initial
-register.) The registration process will:
+<b>Bioinformatics tools face the challenges of irreproducibility.</b> The lack of proper versioning and dependencies management results in a long tail of poorly documented and unusable bioinformatics software tools.
 
-* Build a docker image containing your workflow code
-* Serialize your code and register it with your LatchBio account
-* Push your docker image to a managed container registry
+Latch SDK containerizes and versions the code in the background each time a workflow is registered to the Latch platform. Container images are constructed by parsing the python runtime for imported libraries or defined using a [Dockerfile](https://docs.docker.com/engine/reference/builder/). Similarly, versions are user-specified as any unique plaintext string. This behavior is a strict requirement of the toolchain and gives us remarkable guarantees with respect to code reproducibility, portability and scalability. 
```

Comment:
get rid of "by parsing the python runtime for imported libraries or" (no longer true)